### DISCUSSION TITLE
admin user to see all users files via cli?

### DISCUSSION THREAD

**Discussion by tooptoop4** [drakkan/sftpgo#864] [Q&A] [answered]
when connecting via sftp cli, user1 pushes files to /data/user1/ and can only see those files. user2 pushes files to /data/user2/ and can only see those files. how to make useradmin in sftp cli see both /data/user1/ and /data/user2/ (...and all other users) files?

**Comment by drakkan** [comment, marked as answer]
Set the home to `/data`

**Reply by tooptoop4** [reply]
@drakkan or should it be key_prefix ? since i thought home directory does not show data for s3

**Reply by drakkan** [reply]
> @drakkan or should it be key_prefix ? since i thought home directory does not show data for s3

yes, for s3 is the `key_prefix`
